### DISCUSSION TITLE
How to prevent truncated text in `view = 'mini'`?

### DISCUSSION THREAD

**Discussion by nyngwang** [folke/noice.nvim#273] [Q&A] [answered]
Hi, I got a truncated result as shown in the following image with my `setup` config provided below:

<img src="https://user-images.githubusercontent.com/24765272/208723921-1a73e5b1-07ee-4b32-81db-c9eec834723e.png" width="500">

While it's not a big deal in the image(only the last word `written` is truncated to `writt`). But I once encountered a case where more words were truncated(I didn't capture the image when that happened).

Sorry that I don't know where to start searching to solve this problem. Could anyone help if you know how to setup the config?

Thanks in advance!

## My Config.

```lua
require('noice').setup {
  lsp = {
    -- override markdown rendering so that **cmp** and other plugins use **Treesitter**
    override = {
      ['vim.lsp.util.convert_input_to_markdown_lines'] = true,
      ['vim.lsp.util.stylize_markdown'] = true,
      ['cmp.entry.get_documentation'] = true,
    },
  },
  hover = { enabled = true },
  signature = { enabled = true },
  messages = { view_search = false },
  presets = {
    lsp_doc_border = true,
    long_message_to_split = true,
    inc_rename = true,
  },
}
```

**Comment by folke** [comment]
That doesn't happen for me and can't reproduce.

Can you provide a minimal example? https://github.com/folke/noice.nvim/wiki/Minimal-%60init.lua%60-to-Reproduce-an-Issue

**Reply by nyngwang** [reply]
I just realized that this cannot be reproduced with the minimal config. I think this is probably a cross-plugin issue, since I can still reproduce it with my config. Sorry for my wrong report.

**Comment by nyngwang** [comment, marked as answer]
This is caused by setting `vim.opt.signcolumn = 'yes'` for the floating window of filetype `'noice'` (the filetype of noice-mini).

```lua
vim.api.nvim_create_autocmd({ 'BufWinEnter' }, {
  group = curfile_augroup,
  pattern = '*',
  callback = function ()
    vim.api.nvim_win_set_option(0, 'winhl', '')
    -- vim.opt.signcolumn = 'yes' -- comment/delete this line fixes the problem!
  end,
})
```
